diff -urN -x .build -x .inst -x .sinst -x configure -x 'Makefile.in*' -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x 'mkinstalldirs*' -x autom4te.cache -x '*compile' marbles-1.0.5-orig/CYGWIN-PATCHES/README marbles-1.0.5/CYGWIN-PATCHES/README
--- marbles-1.0.5-orig/CYGWIN-PATCHES/README	1969-12-31 18:00:00.000000000 -0600
+++ marbles-1.0.5/CYGWIN-PATCHES/README	2005-12-14 17:34:59.401625000 -0600
@@ -0,0 +1,137 @@
+marbles
+------------------------------------------
+SDL marbles game
+
+Runtime requirements:
+  libSDL-1.2.9-1
+  libSDL_mixer-1.2.6-1
+  xorg-x11-base-6.8.2.0-1
+
+Build requirements:
+  cygwin-1.5.18-1
+  autoconf2.5-2.59-3
+  automake1.9-1.9.6-1
+  binutils-20050610-1
+  gawk-3.1.5-2
+  gcc-core-3.4.4-1
+  make-3.80-1
+  sed-4.1.4-1
+  libSDL-devel-1.2.9-1
+  libSDL_mixer-1.2.6-1
+
+Canonical website:
+  http://lgames.sourceforge.net/
+
+Canonical download:
+  http://prdownloads.sourceforge.net/lgames/
+
+-------------------------------------------
+
+Build instructions:
+  unpack marbles-1.0.5-X-src.tar.bz2
+    if you use setup to install this src package, it will be
+	 unpacked under /usr/src automatically
+  cd /usr/src
+  ./marbles-1.0.5-X.sh all
+
+This will create:
+  /usr/src/marbles-1.0.5-X-src.tar.bz2
+  /usr/src/marbles-1.0.5-X.tar.bz2
+
+-------------------------------------------
+
+Files included in the binary package:
+
+  /usr/bin/marbles.exe
+  /usr/share/doc/Cygwin/marbles-1.0.5.README
+  /usr/share/doc/marbles-1.0.5/AUTHORS
+  /usr/share/doc/marbles-1.0.5/ChangeLog
+  /usr/share/doc/marbles-1.0.5/COPYING
+  /usr/share/doc/marbles-1.0.5/html/arrow.jpg
+  /usr/share/doc/marbles-1.0.5/html/arrow2.jpg
+  /usr/share/doc/marbles-1.0.5/html/crumble.jpg
+  /usr/share/doc/marbles-1.0.5/html/manual.html
+  /usr/share/doc/marbles-1.0.5/html/marbles.jpg
+  /usr/share/doc/marbles-1.0.5/html/teleport.jpg
+  /usr/share/doc/marbles-1.0.5/INSTALL
+  /usr/share/doc/marbles-1.0.5/NEWS
+  /usr/share/doc/marbles-1.0.5/README
+  /usr/share/doc/marbles-1.0.5/TODO
+  /usr/share/games/marbles/gfx/board.bmp
+  /usr/share/games/marbles/gfx/figure.bmp
+  /usr/share/games/marbles/gfx/f_red.bmp
+  /usr/share/games/marbles/gfx/f_small.bmp
+  /usr/share/games/marbles/gfx/f_white.bmp
+  /usr/share/games/marbles/gfx/f_yellow.bmp
+  /usr/share/games/marbles/gfx/lights.bmp
+  /usr/share/games/marbles/gfx/logo.bmp
+  /usr/share/games/marbles/gfx/marbles.bmp
+  /usr/share/games/marbles/gfx/metal/arrow_d.bmp
+  /usr/share/games/marbles/gfx/metal/arrow_l.bmp
+  /usr/share/games/marbles/gfx/metal/arrow_r.bmp
+  /usr/share/games/marbles/gfx/metal/arrow_u.bmp
+  /usr/share/games/marbles/gfx/metal/background.bmp
+  /usr/share/games/marbles/gfx/metal/barrier_lr.bmp
+  /usr/share/games/marbles/gfx/metal/barrier_ud.bmp
+  /usr/share/games/marbles/gfx/metal/floor.bmp
+  /usr/share/games/marbles/gfx/metal/teleport0.bmp
+  /usr/share/games/marbles/gfx/metal/teleport1.bmp
+  /usr/share/games/marbles/gfx/metal/teleport2.bmp
+  /usr/share/games/marbles/gfx/metal/teleport3.bmp
+  /usr/share/games/marbles/gfx/metal/wall.bmp
+  /usr/share/games/marbles/gfx/metal/wall_crumble.bmp
+  /usr/share/games/marbles/gfx/m_frame.bmp
+  /usr/share/games/marbles/gfx/m_sframe.bmp
+  /usr/share/games/marbles/gfx/stone/arrow_d.bmp
+  /usr/share/games/marbles/gfx/stone/arrow_l.bmp
+  /usr/share/games/marbles/gfx/stone/arrow_r.bmp
+  /usr/share/games/marbles/gfx/stone/arrow_u.bmp
+  /usr/share/games/marbles/gfx/stone/background.bmp
+  /usr/share/games/marbles/gfx/stone/barrier_lr.bmp
+  /usr/share/games/marbles/gfx/stone/barrier_ud.bmp
+  /usr/share/games/marbles/gfx/stone/floor.bmp
+  /usr/share/games/marbles/gfx/stone/teleport0.bmp
+  /usr/share/games/marbles/gfx/stone/teleport1.bmp
+  /usr/share/games/marbles/gfx/stone/teleport2.bmp
+  /usr/share/games/marbles/gfx/stone/teleport3.bmp
+  /usr/share/games/marbles/gfx/stone/wall.bmp
+  /usr/share/games/marbles/gfx/stone/wall_crumble.bmp
+  /usr/share/games/marbles/gfx/tiny.sdlfnt
+  /usr/share/games/marbles/gfx/title.bmp
+  /usr/share/games/marbles/gfx/win_icon.bmp
+  /usr/share/games/marbles/gfx/wood/arrow_d.bmp
+  /usr/share/games/marbles/gfx/wood/arrow_l.bmp
+  /usr/share/games/marbles/gfx/wood/arrow_r.bmp
+  /usr/share/games/marbles/gfx/wood/arrow_u.bmp
+  /usr/share/games/marbles/gfx/wood/background.bmp
+  /usr/share/games/marbles/gfx/wood/barrier_lr.bmp
+  /usr/share/games/marbles/gfx/wood/barrier_ud.bmp
+  /usr/share/games/marbles/gfx/wood/floor.bmp
+  /usr/share/games/marbles/gfx/wood/teleport0.bmp
+  /usr/share/games/marbles/gfx/wood/teleport1.bmp
+  /usr/share/games/marbles/gfx/wood/teleport2.bmp
+  /usr/share/games/marbles/gfx/wood/teleport3.bmp
+  /usr/share/games/marbles/gfx/wood/wall.bmp
+  /usr/share/games/marbles/gfx/wood/wall_crumble.bmp
+  /usr/share/games/marbles/levels/Original
+  /usr/share/games/marbles/sounds/alarm.wav
+  /usr/share/games/marbles/sounds/arrow.wav
+  /usr/share/games/marbles/sounds/click.wav
+  /usr/share/games/marbles/sounds/explode.wav
+  /usr/share/games/marbles/sounds/score.wav
+  /usr/share/games/marbles/sounds/select.wav
+  /usr/share/games/marbles/sounds/stop.wav
+  /usr/share/games/marbles/sounds/teleport.wav
+  /usr/share/man/man6/marbles.6.gz
+  /var/lib/games/marbles.prfs
+
+------------------
+
+Port Notes:
+
+----- version 1.0.5-1 -----
+Initial release for Cygwin.
+
+
+Cygwin port maintained by:
+http://cygwinports.dotsrc.org/
diff -urN -x .build -x .inst -x .sinst -x configure -x 'Makefile.in*' -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x 'mkinstalldirs*' -x autom4te.cache -x '*compile' marbles-1.0.5-orig/CYGWIN-PATCHES/setup.hint marbles-1.0.5/CYGWIN-PATCHES/setup.hint
--- marbles-1.0.5-orig/CYGWIN-PATCHES/setup.hint	1969-12-31 18:00:00.000000000 -0600
+++ marbles-1.0.5/CYGWIN-PATCHES/setup.hint	2005-12-14 17:34:59.401625000 -0600
@@ -0,0 +1,4 @@
+category: Games
+requires: cygwin libSDL libSDL_mixer xorg-x11-base
+sdesc: "SDL marbles game"
+ldesc: "SDL marbles game"
diff -urN -x .build -x .inst -x .sinst -x configure -x 'Makefile.in*' -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x 'mkinstalldirs*' -x autom4te.cache -x '*compile' marbles-1.0.5-orig/src/Makefile.am marbles-1.0.5/src/Makefile.am
--- marbles-1.0.5-orig/src/Makefile.am	2002-08-06 21:13:39.000000000 -0500
+++ marbles-1.0.5/src/Makefile.am	2005-12-14 17:27:25.214125000 -0600
@@ -36,31 +36,41 @@
 marbles_LDADD = @win32_deps@ @mixer_flag@
 man_MANS = marbles.6
 
-DATA_FILES = \
+gfxdir = $(inst_dir)/gfx
+gfx_DATA = \
 gfx/board.bmp gfx/f_red.bmp gfx/f_small.bmp gfx/f_white.bmp \
 gfx/f_yellow.bmp gfx/figure.bmp gfx/lights.bmp gfx/logo.bmp \
 gfx/m_frame.bmp gfx/m_sframe.bmp gfx/marbles.bmp gfx/tiny.sdlfnt  \
-gfx/title.bmp  gfx/win_icon.bmp \
-\
+gfx/title.bmp  gfx/win_icon.bmp
+
+soundsdir = $(inst_dir)/sounds
+sounds_DATA = \
 sounds/alarm.wav sounds/arrow.wav sounds/click.wav sounds/explode.wav \
-sounds/select.wav sounds/stop.wav sounds/teleport.wav sounds/score.wav \
-\
-levels/Original \
-\
+sounds/select.wav sounds/stop.wav sounds/teleport.wav sounds/score.wav
+
+levelsdir = $(inst_dir)/levels
+levels_DATA = levels/Original
+
+stonegfxdir = $(inst_dir)/gfx/stone
+stonegfx_DATA = \
 gfx/stone/arrow_d.bmp gfx/stone/arrow_l.bmp gfx/stone/arrow_r.bmp \
 gfx/stone/arrow_u.bmp gfx/stone/background.bmp gfx/stone/barrier_lr.bmp \
 gfx/stone/barrier_ud.bmp gfx/stone/floor.bmp gfx/stone/teleport0.bmp \
 gfx/stone/teleport1.bmp gfx/stone/teleport2.bmp \
 gfx/stone/teleport3.bmp gfx/stone/wall.bmp \
-gfx/stone/wall_crumble.bmp \
-\
+gfx/stone/wall_crumble.bmp
+
+metalgfxdir = $(inst_dir)/gfx/metal
+metalgfx_DATA = \
 gfx/metal/arrow_d.bmp gfx/metal/arrow_l.bmp gfx/metal/arrow_r.bmp \
 gfx/metal/arrow_u.bmp gfx/metal/background.bmp gfx/metal/barrier_lr.bmp \
 gfx/metal/barrier_ud.bmp gfx/metal/floor.bmp gfx/metal/teleport0.bmp \
 gfx/metal/teleport1.bmp gfx/metal/teleport2.bmp \
 gfx/metal/teleport3.bmp gfx/metal/wall.bmp \
-gfx/metal/wall_crumble.bmp \
-\
+gfx/metal/wall_crumble.bmp
+
+woodgfxdir = $(inst_dir)/gfx/wood
+woodgfx_DATA = \
 gfx/wood/arrow_d.bmp gfx/wood/arrow_l.bmp gfx/wood/arrow_r.bmp \
 gfx/wood/arrow_u.bmp gfx/wood/background.bmp gfx/wood/barrier_lr.bmp \
 gfx/wood/barrier_ud.bmp gfx/wood/floor.bmp gfx/wood/teleport0.bmp \
@@ -72,25 +82,17 @@
 DOC_FILES = manual/manual.html manual/arrow.jpg manual/arrow2.jpg \
 manual/crumble.jpg manual/teleport.jpg manual/marbles.jpg 
 
-EXTRA_DIST= empty.prfs $(DATA_FILES) $(DOC_FILES)
+docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)/html
+doc_DATA = $(DOC_FILES)
+
+EXTRA_DIST= empty.prfs $(gfx_DATA) $(metalgfx_DATA) $(stonegfx_DATA) \
+	$(woodgfx_DATA) $(levels_DATA) $(sounds_DATA) $(DOC_FILES)
 
 install-data-local:
-#directories
-	$(mkinstalldirs) $(DESTDIR)$(inst_dir)
-	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx
-	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx/stone
-	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx/metal
-	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx/wood
-	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/sounds
-	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/levels
-#data files
-	for file in $(DATA_FILES); do\
-	   $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/$$file;\
-	done
 #profiles
 	$(mkinstalldirs) $(DESTDIR)$(prf_dir)
 	if ! test -f $(DESTDIR)$(prf_dir)/marbles.prfs; then \
-	$(INSTALL_DATA) -m 666 empty.prfs $(DESTDIR)$(prf_dir)/marbles.prfs; \
+	$(INSTALL_DATA) -m 666 $(srcdir)/empty.prfs $(DESTDIR)$(prf_dir)/marbles.prfs; \
 	fi;
 
 uninstall-local:
